das/options.go

```diff
@@ -0,0 +1,142 @@
+package das
+
+import (
+	"errors"
+	"fmt"
+	"time"
+)
+
+var (
+	ErrInvalidOption = fmt.Errorf("das: invalid option")
+)
+
+func errInvalidOptionValue(optionName string, value string) error {
+	return fmt.Errorf("%w: value %s cannot be %s", ErrInvalidOption, optionName, value)
+}
```

Comment:
We can remove this once we remove err checking in options

Reply:
It will still be used in `das.Parameters.Validate()`